### ISSUE TITLE
failed to add React Native 0.42.0

### ISSUE THREAD

**Issue by zjhiphop** [infinitered/ignite#912]
---
What's going on?
When i try to create new project with cmd `ignite new xxx`, it always saying `failed to add React Native 0.42.0`

---
Steps to reproduce
```
code  sudo ignite new bolang-rn
-----------------------------------------------
  (                  )   (
  )\ )   (        ( /(   )\ )    *   )
 (()/(   )\ )     )\()) (()/(  ` )  /(   (
  /(_)) (()/(    ((_)\   /(_))  ( )(_))  )\
 (_))    /(_))_   _((_) (_))   (_(_())  ((_)
 |_ _|  (_)) __| | \| | |_ _|  |_   _|  | __|
  | |     | (_ | | .` |  | |     | |    | _|
 |___|     \___| |_|\_| |___|    |_|    |___|
-----------------------------------------------

An unfair headstart for your React Native apps.
https://infinite.red/ignite

-----------------------------------------------

🔥 igniting app bolang-rn
✔ using the Infinite Red boilerplate
✖ failed to add React Native 0.42.0
```
---
`ignite doctor` results:

```
System
  platform           darwin
  arch               x64
  cpu                4 cores                                                         Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz
  directory          /Users/jade/Documents/project/workspace/freelance/bolang/code

JavaScript
  node               7.7.3        /usr/local/bin/node
  npm                4.1.2        /usr/local/bin/npm
  yarn               0.21.3       /usr/local/bin/yarn

React Native
  react-native-cli   2.0.1

Ignite
  ignite             2.0.0-beta.6   /usr/local/bin/ignite

Android
  java               1.8.0_111    /usr/bin/java
  android home       -            /Users/jade/Library/Android/sdk

iOS
  xcode              8.1
```

**Comment by GantMan** [comment]
it might be the name of the project.   Can you give it a name without a hyphen?   Sorry for no friendly error message.
Should be easy to fix if this is the problem.

**Comment by skellock** [comment]
Beta 7 will ship with this fix #902.  React Native doesn't let you create apps with a dash, but we didn't prevent that.

**Comment by mikaelbartlett** [comment]
It also doesn't like ø. So I guess the it needs to be alpha-numeric with ascii chars only?
